### ISSUE TITLE
Many errors when running (statscollector-1.agora.io:6443/events/messages)

### ISSUE THREAD

**Issue by DasGandlaf** [AgoraIO/agora-rtc-web#2]
### AgoraRTC SDK version

4.3.0

### Fail Rate

100%

### Link to minimal reproduction

https://github.com/AgoraIO-Community/VideoUIKit-Web-React

### Steps to reproduce

1. Get uBlock origin and/or Brave browser
2. Open react app
3. Open console
4. Observe the error being thrown when you either have the Brave browser shields up, or the ublock origin extension is active.

### What is expected?

No errors being thrown.

### What is actually happening?

~4 Errors being thrown every second (amounts to thousands in a just a few minutes)
![image](https://github.com/AgoraIO/agora-rtc-web/assets/63406944/07532155-67cc-4611-96c2-5f3193f4e13b)

**Comment by simo-an** [comment]
That is due to uBlock treating stats upload url as AD connection and block it. Please check https://easylist.to/ .

**Comment by DasGandlaf** [comment]
Thank you, yeah thats true. Do you know if there is a way to disable this stats colletction in the code? I would see that if the request fails three times or so, I would stop making the request.

**Comment by simo-an** [comment]
No, there is no way to disable stats collection. We will treat it as an improvement point of web sdk. Thanks.

**Comment by DasGandlaf** [comment]
> No, there is no way to disable stats collection. We will treat it as an improvement point of web sdk. Thanks.

Is there any way to work around this issue or so? I really would not like to go into production with a program which throws thousands of exceptions... I tried to change the source code to not make these requests, but I cannot find where the actual human-readable code is. All I find is the bundled js file of AgoraRTC N-production.js. How can I change it?

**Comment by CREDO23** [comment]
Hello team, i have the same error with the Agora rtc react.

I tried all Agora code examples. The error persists. I need help please :pray: .

**Comment by simo-an** [comment]
> Hello team, i have the same error with the Agora rtc react.
> 
> I tried all Agora code examples. The error persists. I need help please 🙏 .

Okay, we will try to mitigate this problem.

**Comment by LookRain** [comment]
> > Hello team, i have the same error with the Agora rtc react.
> > I tried all Agora code examples. The error persists. I need help please 🙏 .
> 
> Okay, we will try to mitigate this problem.

Thanks, we recently ran into the same issues. Would appreciate if it can be turned off or stopped after multiple failed attempts

**Comment by kzndotsh** [comment]
Any progress? Stats collection should be entirely opt in, in my opinion.
